src/50_Samples_%26_Templates/Product_Listing.html

```diff
@@ -31,12 +31,6 @@
       background: #ECEFF1;
     }
     <%>css/product.css%>
```

Comment:
the search input field is mis-aligned in the listing sample, works in the product sample.